### PR TITLE
Document timestamp delta tokens.

### PR DESCRIPTION
**Pull request by ocreeva**
Document timestamp delta tokens.

### CODE REVIEW

**Review by chackman — APPROVED** [review]

File: api-reference/beta/api/driveitem-delta.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 author: JeremyKelley
 description: "This method allows your app to track changes to a drive and its children over time."
 ms.date: 09/10/2017
-title: Sync the contents of a drive
+title: "driveItem: delta"
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-delta.md:5]: title in line 10 must match with this one. Please edit line 10 as well.

File: api-reference/beta/api/driveitem-delta.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 author: JeremyKelley
 description: "This method allows your app to track changes to a drive and its children over time."
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-delta.md:3]: This description is used by search engines to display results. Please revise to:
Track changes in a drive item and its children over time.

File: api-reference/beta/api/driveitem-delta.md [hunk 3]
```diff
@@ -70,11 +80,13 @@ In addition to the collection of DriveItems, the response will also include one
 | **@odata.nextLink**  | url    | A URL to retrieve the next available page of changes, if there are additional changes in the current set.                                        |
 | **@odata.deltaLink** | url    | A URL returned instead of **@odata.nextLink** after all current changes have been returned. Used to read the next set of changes in the future.  |
 
-## Example (Initial Request)
+## Examples
+
+### Example (Initial Request)
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-delta.md:85]: Could you please use the proper structure:
H3 Example 1: Description
H4 Request
H4 Response

File: api-reference/beta/api/driveitem-delta.md [hunk 4]
```diff
@@ -267,6 +279,69 @@ Content-type: application/json
...
+
+# [JavaScript](#tab/javascript)
+[!INCLUDE [sample-code](../includes/snippets/javascript/get-delta-timestamp-javascript-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+# [Objective-C](#tab/objc)
+[!INCLUDE [sample-code](../includes/snippets/objc/get-delta-timestamp-objc-snippets.md)]
+[!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
+
+# [Java](#tab/java)
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-delta.md:312]: Please remove these snippets and the manually created files for them. These snippets are automatically generated.

File: api-reference/beta/api/driveitem-delta.md [hunk 5]
```diff
@@ -267,6 +279,69 @@ Content-type: application/json
...
+> **Note:** Clients should use the deltaLink provided by `delta` queries when possible, rather than generating their own token. This capability should only be utilized when the deltaLink is not known.
+
+
+#### Request
+
+
+# [HTTP](#tab/http)
+<!-- { "blockType": "request", "name": "get-delta-timestamp", "scopes": "files.read", "tags": "service.graph", "target": "action" } -->
+
+```msgraph-interactive
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-delta.md:297]: Please remove msgraph-interactive and replace by http

File: api-reference/beta/api/driveitem-delta.md [hunk 6]
```diff
@@ -267,6 +279,69 @@ Content-type: application/json
...
+
+Using a timestamp in place of a token is only supported on OneDrive for Business and SharePoint.
+
+> **Note:** Clients should use the deltaLink provided by `delta` queries when possible, rather than generating their own token. This capability should only be utilized when the deltaLink is not known.
+
+
+#### Request
+
+
+# [HTTP](#tab/http)
```
Comment (isvargasmsft) [inline api-reference/beta/api/driveitem-delta.md:294]: Line 294: please remove this line, the snippets are automatically generated.

File: api-reference/beta/includes/snippets/csharp/get-delta-timestamp-csharp-snippets.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+description: "Automatically generated file. DO NOT MODIFY"
```
Comment (isvargasmsft) [inline api-reference/beta/includes/snippets/csharp/get-delta-timestamp-csharp-snippets.md:2]: Please remove this file. These snippets are automatically generated. If you find an error in the generation, please report it to the proper team so they can improve the tool.

File: changelog/Microsoft.FileServices.json [hunk 8]
```diff
@@ -2072,6 +2072,38 @@
       "CreatedDateTime": "2021-09-07T09:20:58.5352861Z",
       "WorkloadArea": "Files",
       "SubArea": ""
+    },
+    {
+        "ChangeList": [{
+            "Id": "F1A3D555-2DA6-4BA2-92C8-7AC9432725A7",
+            "ApiChange": "Method",
```
Comment (isvargasmsft) [inline changelog/Microsoft.FileServices.json:2079]: Please clarify if this is a change in an existing functionality or a new functionality:
Use "Change" if the update affects existing functionality, to call out to developers that they need to update existing apps.
Use "Addition" for new API elements (e.g., resources, methods, enums, enum values, permissions), and for new functionality or permission introduced to existing API elements.

Details here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog?anchor=changelist-record-properties

File: changelog/Microsoft.FileServices.json [hunk 9]
```diff
@@ -2072,6 +2072,38 @@
...
+            "ChangedApiName": "driveItem",
```
Comment (isvargasmsft) [inline changelog/Microsoft.FileServices.json:2080]: ChangedAPIName should be **delta**, as it is the element being changed.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @chackman , 

Thank you very much for taking ownership of this PR and for your work. 

Please apply my comments in all documents in this PR where applicable. 

Feel free to ping me if you have any questions. 

Here to help, 

Isaac Vargas

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**